src/components/comment/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Fragment } from 'react';
+
+// Components
+import CommentContent from './CommentContent';
+
+// MUI
+import Grid from '@material-ui/core/Grid';
+import withStyles from '@material-ui/core/styles/withStyles';
+import style from '../../style/style';
+
+const Comment = ({ classes = {}, comment = {}, index, length }) => {
+  const createVisibleSeparator = () => {
+    if (index !== length - 1) {
+      return <hr className={classes.visibleSeparator} />;
+    }
+    return <hr className={classes.invisibleSeparator} />;
+  };
+  return (
+    <Grid container>
+      <Fragment>
+        <Grid item sm={12}>
+          <CommentContent classes={classes} comment={comment} />
+        </Grid>
+        {createVisibleSeparator()}
```

Comment:
Instead of using a separator, why not some kind of Card component? Here's an example of one: https://developer.microsoft.com/en-us/fabric#/controls/web/card

It's considered cleaner to assign the look/feel of a row item in a list to the actual row item instead of to some notion of the space in between. You don't have to deal with fenceposting if you just put the comments in a card/panel/whatever container.

Reply:
fixed